File: 3227-find-missing-and-repeated-values/find-missing-and-repeated-values.java
class Solution {
    public int[] findMissingAndRepeatedValues(int[][] grid) {
        int n = grid.length;
        boolean[] check = new boolean[n*n+1];
        check[0] = true;
        int[] res = new int[2];
        for (int i=0; i<n; i++) {
            for (int j=0; j<n; j++) {
                if (check[grid[i][j]] == true)  res[0] = grid[i][j];
                check[grid[i][j]] = true;
            }
        }
        for (int i = 0; i < n*n+1; i++) {
            if (check[i] == false) {
                res[1] = i;
                break;
            }
        }
        return res;
    }
}

File: 3227-find-missing-and-repeated-values/find-missing-and-repeated-values.py
class Solution:
    def findMissingAndRepeatedValues(self, grid: List[List[int]]) -> List[int]:
        n = len(grid)
        check = [False] * (n*n+1)
        check[0] = True
        res = [0, 0]
        for i in range(n):
            for j in range(n):
                if check[grid[i][j]]:
                    res[0] = grid[i][j]
                check[grid[i][j]] = True
        res[1] = check.index(False)
        return res